### ISSUE TITLE
Nice!

### ISSUE THREAD

**Issue by DeepRobin** [ngirot/BruteForce#1]
Hello, this is a nice brute force repository, i love it. I use it currently in my repository "Controller" for a sshBruteForce command.

**Comment by ngirot** [comment]
Hi,

Thank you for your comment, I'm glad this code helped someone !

I did take a look on your command repo, and I think there is a little misunderstanding about the second parameter of NewWorderAlphabet function.
This "step" parameter is used to choose the increment between two words. For example with an alphabet "a-z", the "Next" function will return : 
With a step parameter at 1:
`a -> b -> c -> d -> e`
With a step parameter at 2:
`a -> c -> e -> g`
With a step parameter at 3:
`a -> d -> g -> j`

It's very useful for me because i have multiple go routines generating a word and hashing it, so each go routines will pick a different word each time.
If i have two go routine, I create the two worder with this parameter
`words.NewWorderAlphabet(words.DefaultAlphabet(), 2, 0)`
`words.NewWorderAlphabet(words.DefaultAlphabet(), 2, 1)`

Then the first will return in order "a c e" ad the second "b d f"

Your code looks more "sequential", so if you use a value different from 1, some words will never been return by the worder.

I'm gonna change my code to make this more explicit (maybe in January, I can't work on it right now).

TLDR : I think you should replace your line 

`var worder = words.NewWorderAlphabet(words.DefaultAlphabet(), numberOfChans, 0)`

by

`var worder = words.NewWorderAlphabet(words.DefaultAlphabet(), 1, 0)`

**Comment by RLEXBuilding** [comment]
Thank you :)
If you want you can contribute to my repository, too
